cds_ils/static_pages/search_guide.html

```diff
@@ -2,53 +2,53 @@
 
 <h3>Simple search (one or multiple terms)</h3>
 
-<p><strong>Example:</strong> <a href="#"><code>dark matter</code></a></p>
+<p><strong>Example:</strong> <a href="https://catalogue.library.cern/search?page=1&size=21&q=dark%20matter"><code>dark matter</code></a></p>
```

Comment:
The domain name cannot be hardcoded because we have a local dev instance, DEV, Sandbox and PROD environments.
Can you omit the domain name and simply put `/search?...`?